stock_request/models/stock_request.py

```diff
@@ -48,6 +44,7 @@ def _get_expected_date():
     expected_date = fields.Datetime(
         "Expected Date",
         index=True,
+        default=lambda self: fields.Datetime.now(),
```

Comment:
@petrus-v I would have put this in a function in order to allow override.

Reply:
I agree there is no reason for this change !...

Reply:
hum.. I mean I'll use the existing method !

Reply:
done